geckomat/get_enzyme_data/loadBRENDAdata.m

```diff
@@ -1,22 +1,23 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
- function [KCATcell, SAcell] = loadBRENDAdata
-     KCAT_file      = '../../databases/max_KCAT.txt';
-     SA_file        = '../../databases/max_SA.txt';
-     MW_file        = '../../databases/max_MW.txt';
+ function [KCATcell, SAcell] = loadBRENDAdata(modelAdapter)
+     KCAT_file      = modelAdapter.getFilePath('max_KCAT.txt');
+     SA_file        = modelAdapter.getFilePath('max_SA.txt');
+     MW_file        = modelAdapter.getFilePath('max_MW.txt');
```

Comment:
These files are not model specifric, so should be kept in the `databases` folder?

Reply:
> These files are not model specific, so should be kept in the `databases` folder?

I agree, they are `databases`

Reply:
Ah, I see that they are model specific, because they only include human kcat values. If that is necessary, it would be better to provide a function that can load the full generic database files and subset those for the desired organism, instead of providing more model-specific files.

Reply:
But this databases are also used to fill those reactions with the proteins information from other organism in case it is not reported for that organism, or if the model is over constrained

Reply:
Indeed, it does not replace the generic database files.

Reply:
Shall we still use the files `max_KCAT.txt`, `max_SA.txt` and `max_MW.txt` generated 4 years ago? And Brenda has already changed the manner to access the data, see [this](https://www.brenda-enzymes.org/download.php). The python part in GECKO1&2 that retrieves enzyme information from Brenda is obsolete.

Reply:
We can repurpose the DLKcat code that gathers this data?

Reply:
This kind of update of BRENDA is after DLKcat. I have code for gather and preprocess this data , but it is just functional not so clean.

Reply:
We would just run this locally every now and then (once a year?) and update the files in the repo, similar as KEGG-trained HMMs in RAVEN. So it's fine if the pipeline is not so clean at the moment, as long as the output is consistent. We can then write a function that can take the output (probably json?) and make the `max_*.txt` files.